samples/react/.eslintrc

```diff
@@ -0,0 +1,42 @@
+{
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "sourceType": "module",
+    "allowImportExportEverywhere": false,
+    "codeFrame": false
+  },
+  "extends": [
+    "airbnb",
+    "prettier"
+  ],
+  "env": {
+    "browser": true,
+    "jest": true
+  },
+  "rules": {
+    "max-len": [
+      "error",
+      {
+        "code": 100
+      }
+    ],
+    "react/button-has-type": [
+      "off"
```

Comment:
[question] Any reason why this is set to off? It's best practice to specify a type on a <button>.

Reply:
Yes, I agree with this. The problem is that for some reason ESLint does not like it to be added dynamically, It wants to see an actual type there. I tried a few ways but was not successful.

Reply:
Ah ok, gotcha.